### PR TITLE
Added support for adding additional python project inside the wheel

### PR DESCRIPTION
**Pull request by neel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,9 @@ add_library(mylib ...)
 #  Python module dependencies (requirements.txt content)
 # SCRIPTS
 #  Additional scripts that should be part of the wheel.
+# PYTHON_PACKAGE_DIRS
+#  Additional python projects which could be wrappers around the
```
Comment (josephbirkner) [inline README.md:48]: I would rephrase this: "Paths to directories of additional Python packages, which are bundled with the wheel. This could for example contain pybind11 binding module wrappers."

Comment (neel) [reply README.md]: Okay. did that. However I am using it with Boost.Python not pybind11. I guess it doesn't matter.

Comment (josephbirkner) [reply README.md]: Awesome, good to know. Thanks!

**Review by josephbirkner — APPROVED** [review]
